platform/msm_shared/scm.c

```diff
@@ -1312,6 +1312,7 @@ static int scm_call2_atomic(uint32_t svc, uint32_t cmd, uint32_t arg1, uint32_t
 	return ret;
 }
 
+#if PLATFORM_USE_SCM_DLOAD
```

Comment:
Is this really necessary?

Reply:
Won't compile otherwise..

Reply:
I wonder if https://github.com/msm8916-mainline/lk2nd/commit/5e72e49ea0e027d4d526581c97d3a9437326b36a fixes it

Reply:
It does, that's what I get for not syncing before developing...